etc/kayobe/ansible/roles/nova-compute-drain/tasks/instance-info.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Query instances
+  command: >
+    {{ nova_compute_drain_venv }}/bin/openstack
+    server list --host {{ ansible_facts.fqdn }}
+    --all-projects
+    --format json
+  register: instances
+  environment: "{{ openstack_auth_env }}"
+  vars:
+    ansible_host: "{{ hostvars[nova_compute_drain_delegate_host].ansible_host }}"
```

Comment:
Does it need delegate_to?

Reply:
Hmm, yep looks like it. Not sure, how that got lost! I'll re-add.